pkg/build/registry/buildconfig/rest_test.go

```diff
@@ -386,7 +392,9 @@ func TestBuildRESTValidatesUpdate(t *testing.T) {
 				Strategy: api.BuildStrategy{
 					Type: api.STIBuildStrategyType,
 					STIStrategy: &api.STIBuildStrategy{
-						Image: "builder/image",
+						From: kapi.ObjectReference{
```

Comment:
There need to be tests that validate that Image still works - most of the places where you've added From should be duplicated and continue to test Image as well.